### PR TITLE
[database][admin 1/x] Adds HackerReviews table 

### PR DESCRIPTION
**Pull request by saresend**
## What 
This adds the hacker Review table so that we have a database table from which to work off of

Begins work for #95

### CODE REVIEW

File: api/admin.js [hunk 1]
```diff
@@ -19,4 +19,16 @@ router.put("/:email", async (req, res) => {
...
 });
 
+router.get("/reviews", async (req, res) => {
+  console.log("we get here");
+  try {
+    const reviews = await models.HackerReview.findAll();
+    console.log(reviews);
+    return res.json({ reviews: reviews });
+  } catch (e) {
+    console.log("WE FUCKED UP BOYS");
```
Comment (WilhelmWillie) [inline api/admin.js]: lmaoooo

File: api/database/migrations/20191115010855-add-review-table.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("HackerReviews", {
```
Comment (WilhelmWillie) [inline api/database/migrations/20191115010855-add-review-table.js]: lit -- needs a corresponding model definition in `api/database/models` but lgtm after that

File: api/admin.js (hunk 1 above)
Comment (saresend) [reply api/admin.js]: Y I K E S

File: api/database/migrations/20191115010855-add-review-table.js (hunk 2 above)
Comment (saresend) [reply api/database/migrations/20191115010855-add-review-table.js]: should be in the next PR, just wanna make sure it all seems good to you

Comment (WilhelmWillie) [reply api/database/migrations/20191115010855-add-review-table.js]: lit yeah this lgtm

**Pull request merged**